utils/utils.py

```diff
@@ -53,7 +53,7 @@ def sample_normal(mean, log_var):
             latent_dim)
     """
     std = tf.math.exp(0.5 * log_var)
-    epsilon = tf.random.normal(std)
+    epsilon = tf.random.normal(tf.shape(std)) # TODO is shape the correct thing to use here?
```

Comment:
What do you mean?

Reply:
I think it is ok, but I wanted to double check with the paper when I had a chance

Reply:
Checked: this is correct